commit fde1ad580573bbcd40d8a2148ebeaa3589f639bf (refs/changes/85/2585/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2017-09-15T23:20:12+00:00 (2 years, 1 month ago)
    
    MORAY-429 Update "vasync" dependency to get newer "verror"
    MORAY-434 Update Moray server to "fast" 2.2.4
    Reviewed by: David Pacheco <dap@joyent.com>
    Approved by: David Pacheco <dap@joyent.com>

diff --git a/package.json b/package.json
index 37da867..f150d9b 100644
--- a/package.json
+++ b/package.json
@@ -14,7 +14,7 @@
         "crc": "0.2.1",
         "dtrace-provider": "0.2.8",
         "deep-equal": "0.0.0",
-        "fast": "2.2.3",
+        "fast": "2.2.4",
         "ip6addr": "0.1.1",
         "jsprim": "1.4.0",
         "kang": "1.2.0",
@@ -30,7 +30,7 @@
         "pg-parse-float": "0.0.1",
         "pooling": "0.4.5",
         "posix-getopt": "1.0.0",
-        "vasync": "1.4.2",
+        "vasync": "1.6.4",
         "verror": "1.3.7",
         "xtend": "2.1.2"
     },
